express-delete/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+
+let grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.use('/api/grades/:id', (req, res) => {
```

Comment:
This should use `app.delete` so it is only called when the request is made with the `DELETE` method.